src/mmw/js/src/draw/views.js

```diff
@@ -172,12 +172,18 @@ var PlaceMarkerView = Marionette.ItemView.extend({
 });
 
 function changeOutlineLayer(endpoint, tableId, model) {
-    var map = App.getLeafletMap();
+    var map = App.getLeafletMap(),
+        ofg = model.get('outlineFeatureGroup');
 
-    if (endpoint && tableId != undefined) {
+    // If the outlines are already present, there is nothing more to do.
+    if (ofg.getLayers().length !== 0) {
```

Comment:
Should we also check that the selected layer is the layer that is already on the map? (I know we only have one layer at the moment).